code/game/objects/items/storage/boxes.dm

```diff
@@ -863,16 +863,77 @@
 	for(var/i in 1 to 6)
 		new /obj/item/ammo_casing/shotgun/beanbag(src)
 
-/obj/item/storage/box/actionfigure
+/obj/item/storage/box/figurines
 	name = "box of action figures"
-	desc = "The latest set of collectable action figures."
+	desc = "The latest set of collectable figurines."
 	icon_state = "box"
 
-/obj/item/storage/box/actionfigure/PopulateContents()
+/obj/item/storage/box/figurines/PopulateContents()
 	for(var/i in 1 to 4)
-		var/randomFigure = pick(subtypesof(/obj/item/toy/figure))
+		var/randomFigure = pick(pick(subtypesof(/obj/item/toy/figure)),
+			pick(subtypesof(/obj/item/toy/prize)),
+			pick((subtypesof(/obj/item/toy/talking) - typesof(/obj/item/toy/talking/codex_gigas))),
+			pick((subtypesof(/obj/item/toy/plush) - typesof(/obj/item/toy/plush/goatplushie/angry/kinggoat))))
 		new randomFigure(src)
 
+/obj/item/storage/box/decorations
+	name = "box of decorations"
+	desc = "A fun decoration to liven up the room!"
+	icon_state = "box"
+
+/obj/item/storage/box/decorations/PopulateContents()
+	var/randomDecoration = pick(/obj/item/storage/crayons,
+		/obj/item/stack/tile/fakespace/loaded,
+		/obj/item/stack/tile/fakepit/loaded,
+		/obj/item/stack/tile/eighties/loaded,
+		pick(subtypesof(/obj/item/grenade/chem_grenade/glitter)))
+	new randomDecoration(src)
+
+/obj/item/storage/box/gamergear
+	name = "box of gamer gear"
+	desc = "This is what we like to call some pro-gamer gear!"
+	icon_state = "box"
+
+/obj/item/storage/box/gamergear/PopulateContents()
+	var/randomGamergear = pick(/obj/item/storage/box/fakesyndiesuit,
+        /obj/item/storage/belt/military/snack,
+        /obj/item/clothing/under/syndicate/tacticool,
+        /obj/item/clothing/shoes/wheelys,
+        /obj/item/clothing/shoes/kindleKicks,
+        /obj/item/clothing/glasses/trickblindfold)
+	new randomGamergear(src)

```

Comment:
honestly, just drop the box for this result and spawn the gamer gear directly. it doesn't really make much sense to dispense a box that holds a single item anyway.

Reply:
How would you recommend I go about the change you are recommending. If I put the `pick` in the loot pool it init's once, it will only spawn the first `pick` anytime the loot pool is used.

Reply:
Rather than a box, have there be an item, that upon spawning spawns the target item, and qdels itself.